src/pycdr/setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = pycdr
+version = 0.1.5
+author = Thijs Miedema
+author_email = thijs.miedema@adlinktech.com
+description = Python CDR serialization
+url = https://github.com/eclipse-cyclone/cyclonedds-python
+project_urls =
+    Bug Tracker = https://github.com/eclipse-cyclone/cyclonedds-python/issues
+classifiers =
+    Development Status :: 4 - Beta
+    License :: OSI Approved :: Eclipse Public License 2.0 (EPL-2.0)
```

Comment:
Shouldn't EDLv1 be mentioned as well?

Reply:
It really should, but it is not in the officially supported classifier list. I am looking into how to get it on there.

Reply:
I wonder if you cannot just use BSD-3-Clause instead? It's the same license and that's what we mention in the headers(?)

Reply:
`SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause` that's what's in the top of our headers...

Reply:
The full list is [here](https://pypi.org/classifiers/), BSD-3 is also not on there. I am wondering, since these classifiers are just strings, if I can just add new ones. I will run a test on the test-pypi website.

Reply:
I found instructions in the Pypi FAQ and filed an issue [here](https://github.com/pypa/trove-classifiers/issues/70) for it to be added.